chainerx_cc/chainerx/routines/indexing.h

```diff
@@ -27,19 +42,7 @@ Array At(const Array& a, const std::vector<ArrayIndex>& indices);
 // `axis` must be within [0, b.ndim()).
 // `indices` must have dtype kind of either kInt or kUInt.
 
-class AddAtOp : public Op {
-public:
-    static const char* name() { return "AddAt"; }
-
-    virtual void Call(const Array& a, const Array& indices, int8_t axis, const Array& b, const Array& out);
-
-protected:
-    virtual void Impl(const Array& a, const Array& indices, int8_t axis, const Array& b, const Array& out) = 0;
-};
-
-inline void AddAt(const Array& a, const Array& indices, int8_t axis, const Array& b, const Array& out) {
-    a.device().backend().CallOp<AddAtOp>(a, indices, axis, b, out);
-}
+Array AddAt(const Array& a, const Array& indices, int8_t axis, const Array& b);
```

Comment:
I think the comment should be updated not to mention `out`.

Reply:
Addressed in https://github.com/chainer/chainer/pull/6789.